tests/testthat/test-workflow-bbi.R

```diff
@@ -305,5 +305,160 @@ withr::with_options(list(
     run_times <- model_summaries(mods) %>% check_run_times(.wait = FALSE) %>% suppressWarnings()
     expect_equal(dim(run_times), c(3, 3))
   })
+
+  describe("run_nmtran", {
```

Comment:
These tests are dependent on being on Metworx, but they're relatively fast compared to the other tests in this file. It would be nice to have these NM-TRAN tests in a separate file to make it easier to execute independently.

(I would prefer if the new test file didn't follow this one's setup code and instead moves to using withr to set up more isolated test directories rather than working under the repo's directory. But I won't object if you end up going that way.)